### PR TITLE
BCROS: MHR Tile Updates

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#15132

*Description of changes:*
* Addition of MHR Tile under BC Registries Accounts
* Text updates to BC ONLINE Mhr tile
* Addition of Mhr enable FF
* Addition of Mhr Beta chip FF
* Updated env/vault url variable

### CODE REVIEW

File: components/BCRegistriesAccount.vue [hunk 1]
```diff
@@ -80,6 +80,41 @@
         </v-card-text>
       </v-card>
 
+      <!-- Business Search -->
```
Comment (cameron-eyds) [inline components/BCRegistriesAccount.vue:83]: Existing tile, reordered according to design.

**Review by doug-lovett — APPROVED** [review]
Looks good.

**Review by pwei1018 — APPROVED** [review]

**Comment by pwei1018** [comment]
> /gcbrun

this doesn't work right now.

File: components/BCRegistriesAccount.vue [hunk 2]
```diff
@@ -416,6 +447,14 @@ export default class BCRegistriesAccount extends Vue {
     return getFeatureFlag('bcregistry-ui-bca-enabled') as boolean
   }
 
+  get showMhrFlag (): boolean {
+    return getFeatureFlag('bcregistry-ui-mhr-enabled') as boolean
```
Comment (seeker25) [inline components/BCRegistriesAccount.vue:451]: Are these added in LD?

**Review by seeker25 — APPROVED** [review]

File: components/BCRegistriesAccount.vue (hunk 2 above)
Comment (cameron-eyds) [reply components/BCRegistriesAccount.vue:451]: This was pre existing yessir, but wasn't in use anywhere that Patrick or myself could see. 

The beta flag was net new and added though, afaik.

**Pull request merged**